account_invoice_ubl/models/account_move.py

```diff
@@ -248,6 +280,8 @@ def generate_invoice_ubl_xml_etree(self, version="2.1"):
         xml_root = etree.Element("Invoice", nsmap=nsmap)
         self._ubl_add_header(xml_root, ns, version=version)
         self._ubl_add_order_reference(xml_root, ns, version=version)
+        self._ubl_add_buyer_reference(xml_root, ns, version=version)
```

Comment:
In the [XSD document specification](https://github.com/sunflowerit/edi/blob/13.0-add-account_invoice_ubl_peppol/base_ubl/data/xsd-2.2/maindoc/UBL-Invoice-2.2.xsd#L407) which the unit test checks for (and the online PEPPOL validators as well) it's specified that BuyerReference should come before OrderReference. So this one should go one line up, and then it might pass the test. If not, we still might have to shuffle things around a bit.

Reply:
I think your were right about this coming before order all seems well now got a green validation after uploading attached sample.
![image](https://user-images.githubusercontent.com/10249832/158906631-78335bdd-79d3-4692-8fa2-b01dee0e88b8.png)